src/worldofzuul/environment/Room.java

```diff
@@ -130,7 +138,7 @@ public void printItems(){
         else{
             System.out.println("The room contains: ");
             for(Item item : items){
-                System.out.print(item.getName()+" ");
+                System.out.print(item.getName()+", ");
```

Comment:
Hvis det bliver ændret til en println vil der ikke komme et komma når der kun er et item, da kommaet kan slettes.
Da det bliver printet som en liste

Reply:
fixed in: 1c37c525cdf54a49a8c73ae75e05ad868fbfe0d6